torch/_logging/_internal.py

```diff
@@ -126,6 +126,7 @@ def set_logs(
     bytecode=None,
     aot_graphs=None,
     aot_joint_graph=None,
+    ddp_graphs=None,
```

Comment:
An oss contributor added nice comments for each arg in a recent PR can you update those too to explain to users what these are?